### DISCUSSION TITLE
Vlnplot contradicts the avg_log2FC

### DISCUSSION THREAD

**Discussion by cyta1a3** [satijalab/seurat#6396] [Q&A]
I have used FindAllMarkers() to analyze my data， and the Acta2 expression level in cluster 5 is low according to the avg_log2FC， but when I used VlnPlot() to show the difference， vlnplot shows that cluster 5 has a very high expression level， which confused me a lot.
I wonder why this happened， and which one I should trust. Thanks!
![image](https://user-images.githubusercontent.com/76946452/189488864-d168d00f-8dbb-4156-91dd-bec2fb655796.png)
![image](https://user-images.githubusercontent.com/76946452/189489025-6ec713a8-adda-4e39-9198-b7dfee251780.png)

**Comment by f6v** [comment]
> Acta2 expression level in cluster 5 is low according to the avg_log2FC

avg_log2FC is the average log fold change in a cluster compared to all other. $2^{0.837}=1.7863317$. This means that it's unregulated in 5 compared to others.
